misc/python/materialize/cli/mzcompose.py

```diff
@@ -33,7 +34,19 @@
 
 def main(argv: List[str]) -> int:
     # Lightly parse the arguments so we know what to do.
-    args, unknown_args = ArgumentParser().parse_known_args(argv)
+    parser = ArgumentParser()
+    args, unknown_args = parser.parse_known_args(argv)
+    if args.help:
+        parser.print_help()
+        print()
+        print(
+            "Additionally, mzcompose is a thin wrapper around docker-compose,\n"
+            "All of the following arguments will be forwarded:"
+        )
+        print()
+        subprocess.run(["docker-compose", "--help"])
+        return 0
```

Comment:
This will be the wrong return code if `docker-compose` does not exist or the command otherwise fails.

Reply:
ah yeah, I forgot that subprocess.run defaults to `check=False`, while I agree it's worth fixing I'm not sure how huge of a deal this is, since `--help` shouldn't be run in automation.